src/App.js

```diff
@@ -1,12 +1,15 @@
 import { BrowserRouter } from 'react-router-dom';
 import Routes from './routes';
 import ThemeCustomization from './themes';
+import NavBar from './components/Navbar';
 
 // ==============================|| APP - THEME, ROUTER, LOCAL  ||============================== //
 
 const App = () => (
   <ThemeCustomization>
+      
```

Comment:
can remove this new line

Reply:
Addressed by [00dd588](https://github.com/cse-mosip/Exam-Attendance-UI/pull/57/commits/00dd588c1c722aa5f819c9a11f629962c6ac8e29)